packages/account/src/Sections/Security/ConnectedApps/connected-apps-info.scss

```diff
@@ -0,0 +1,6 @@
+.connected-apps-list {
```

Comment:
we already have a local scss file for `connected-apps` component. please move this to that file.
`packages/account/src/Sections/Security/ConnectedApps/connected-apps.scss`

Reply:
Done